protoc-gen-go/testdata/grpc/grpc.go

```diff
@@ -0,0 +1,470 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
What is this file? How is this different from `protoc-gen-go/testdata/grpc/grpc.pb.go`?

Reply:
Yes, this was a mistake on my part. Got added in by mistake. Fixed.